test/append.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var eq = require('./utils').eq;
+var S = require('..');
+
+describe('append', function() {
```

Comment:
That is something that should/could be added to a sanctuary eslint config. I'll note that for later.